deploy/roles/postgresql-client/defaults/main.yml

```diff
@@ -8,6 +8,16 @@ postgres_install_repository: True
 postgres_packages:
   - postgresql95
   - postgresql95-devel
+  - htop
+  - cups
+  - hplip
+  - gutenprint
+  - foomatic
+  - udev
+  - firefox
+  - system-config-printer
+  - postgresql-odbc
+  - postgresql95-odbc
```

Comment:
@olivermel We don't want all of these packages to be installed on every server that uses the postgresql-client role, just the research environment.  Let's keep postgresql95-odbc in here, remove postgresql-odbc completely, and move the rest of these new ones directly into the Research Environment playbook?